packages/components/src/ViewTransactionLink.tsx

```diff
@@ -1,35 +1,59 @@
 import * as React from "react";
 import { TxHash } from "@joincivil/core";
-import styled from "styled-components";
+import styled, {withTheme} from "styled-components";
 import { colors, fonts } from "./styleConstants";
 import { NorthEastArrow } from "./icons";
 
 export interface ViewTransactionLinkProps {
   txHash: TxHash;
   network: string;
+  text?: string;
+  theme: LinkTheme;
+}
+
+export interface ViewTransactionLinkState {
+  hovered: boolean;
 }
 
 export interface LinkTheme {
+  linkColorHover: string;
   linkColor: string;
 }
 
+const defaultProps = {
+  theme: {
+    linkColor: colors.accent.CIVIL_BLUE,
+    linkColorHover: colors.accent.CIVIL_BLUE_FADED,
+    sanserifFont: fonts.SANS_SERIF,
+  },
+};
+
 const StyledLink = styled.a`
   text-decoration: none;
-  font-family: ${fonts.SANS_SERIF};
+  font-family: ${props => props.theme.sanserifFont};
   color: ${props => props.theme.linkColor};
 `;
 
-StyledLink.defaultProps = {
-  theme: {
-    linkColor: colors.accent.CIVIL_BLUE,
-  },
-};
+StyledLink.defaultProps = defaultProps;
 
-export const ViewTransactionLink = (props: ViewTransactionLinkProps): JSX.Element => {
-  const baseUrl = props.network === "rinkeby" ? "https://rinkeby.etherscan.io/tx/" : "https://etherscan.io/tx/";
-  return (
-    <StyledLink target="_blank" href={`${baseUrl}${props.txHash}`}>
-      view transaction <NorthEastArrow />
-    </StyledLink>
-  );
-};
+class ViewTransactionLinkComponent extends React.Component<ViewTransactionLinkProps, ViewTransactionLinkState> {
+  constructor(props: ViewTransactionLinkProps) {
+    super(props);
+    this.state = {
+      hovered: false,
+    };
+  }
+  public render(): JSX.Element {
+    const baseUrl = this.props.network === "rinkeby" ? "https://rinkeby.etherscan.io/tx/" : "https://etherscan.io/tx/";
+    return (
+      <StyledLink onMouseOver={() => this.setState({hovered: true})} onMouseLeave={() => this.setState({hovered: false})} target="_blank" href={`${baseUrl}${this.props.txHash}`}>
+        {this.props.text || "view transaction"} <NorthEastArrow color={this.state.hovered ? this.props.theme.linkColorHover : this.props.theme.linkColor} />
```

Comment:
Dang, this color stuff can't be done w/ just CSS? 

I think something this might work, if `NorthEastArrow` is a `StyledComponent`, but not a dealbreaker if this isn't possible.

```
const StyledLink = styled.a`
  &:hover ${NorthEastArrow} { 
    fill: ...,
    stroke: ...
  }
`
```

https://www.styled-components.com/docs/advanced#referring-to-other-components